charts/kube-prometheus-stack/values.yaml

```diff
@@ -1853,6 +1853,99 @@ prometheus-node-exporter:
     ##
     pspEnabled: false
 
+## Manages Prometheus Operator CRD installation and upgrade
+##
+crds:
+  enabled: true
+
+  managedResources:
+  - monitoring.coreos.com_alertmanagerconfigs.yaml
+  - monitoring.coreos.com_alertmanagers.yaml
+  - monitoring.coreos.com_podmonitors.yaml
+  - monitoring.coreos.com_probes.yaml
+  - monitoring.coreos.com_prometheusagents.yaml
+  - monitoring.coreos.com_prometheuses.yaml
+  - monitoring.coreos.com_prometheusrules.yaml
+  - monitoring.coreos.com_scrapeconfigs.yaml
+  - monitoring.coreos.com_servicemonitors.yaml
+  - monitoring.coreos.com_thanosrulers.yaml
+
+  image:
+    registry: docker.io
+    repository: bitnami/kubectl
+    tag: 1.26.3
+    sha: ""
+    pullPolicy: IfNotPresent
+
+  ## Force conflicts resolution when running kubectl apply
+  ##
+  forceConflicts: false
+
+  networkPolicy:
+    ## Enable creation of NetworkPolicy resources.
+    ##
+    enabled: false
+
+  ## Provide a priority class name to the CRD install job
+  ##
+  priorityClassName: ""
+
+  sourceUrl: https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/{{ $.Chart.AppVersion }}/example/prometheus-operator-crd/
+
+  ## Service account for the CRD install job to use.
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/
+  ##
+  serviceAccount:
+    create: true
+    name: ""
+    annotations: {}
+
+  resources: {}
+
+  ## Annotations to add to the CRD install pod
+  ##
+  podAnnotations: {}
+
+  ## Define which Nodes the Pods are scheduled on.
+  ## ref: https://kubernetes.io/docs/user-guide/node-selection/
+  ##
+  nodeSelector: {}
+
+  ## Tolerations for use with node taints
+  ## ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+  ##
+  tolerations: []
+  # - key: "key"
+  #   operator: "Equal"
+  #   value: "value"
+  #   effect: "NoSchedule"
+
+  ## Assign custom affinity rules to the CRD install pod
+  ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
+  ##
+  affinity: {}
+    # nodeAffinity:
+    #   requiredDuringSchedulingIgnoredDuringExecution:
+    #     nodeSelectorTerms:
+    #     - matchExpressions:
+    #       - key: kubernetes.io/e2e-az-name
+    #         operator: In
+    #         values:
+    #         - e2e-az1
+    #         - e2e-az2
+
+  ## SecurityContext holds pod-level security attributes and common container settings.
+  ## This defaults to non root user with uid 2000 and gid 2000. *v1.PodSecurityContext  false
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
+  ##
+  securityContext:
```

Comment:
If this could be made compatible with the restricted profile by default that would be great.